src/binary32.rkt

```diff
@@ -2,11 +2,7 @@
 
 ;; binary32 builtin plugin
 
-(require math/base math/bigfloat math/flonum math/special-functions)
-(require (submod "syntax/syntax.rkt" internals)
-         (submod "interface.rkt" internals)
-         "syntax/syntax.rkt" "common.rkt"
-         "errors.rkt")
+(require herbie/plugin math/flonum math/bigfloat herbie/errors)
```

Comment:
When you require `herbie/plugin` and `herbie/errors` this way instead of with a path, it becomes more annoying to use locally (you need to `raco pkg install` it, and then sometimes you need to reinstall for obscure reasons). What's worse is—if you have two Herbie directories on your machine, it might load the wrong code. We should use the path.